Screen_OLED_Test/Screen_OLED_Test.ino

```diff
@@ -1,18 +1,20 @@
 #include "OLEDScreen.h"
 #define modeSwitchButton 6
 
-bool direction = true;
+bool directionGasReverse = true; // Forward
+bool directionTurn = true; // Left
 
 void setup(void)
 {
-  initIOModule(modeSwitchButton, direction);
+  initIOModule(modeSwitchButton, directionGasReverse, directionTurn, true);
 }
 
 void loop(void)
 {
   showTime();
   timeSave();
   drawDisplayBitmaps();
+  // testBitmaps(); // Debugging/Test
```

Comment:
Also when you have the condition with the if you don't need to comment this line out